### PR TITLE
PE-4469: UI overflow issues in various scenarios

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/6k97n247pc7ho

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -64,6 +64,10 @@
           <category android:name="android.intent.category.BROWSABLE" />
           <data android:scheme="https" />
         </intent>
+        <intent>
+          <action android:name="android.intent.action.SENDTO" />
+          <data android:scheme="mailto" />
+        </intent>
```
Comment (matibat) [inline android/app/src/main/AndroidManifest.xml:70]: This change is out of scope, right?

Comment (matibat) [reply android/app/src/main/AndroidManifest.xml:70]: Ahh. It's related to the export logs one - https://github.com/ardriveapp/ardrive-web/pull/1310/files#diff-58f53cf52a6fb0e122f9ce2271ab990f201988ecc3bdd30f71dbbcf4109aa201R460

File: lib/utils/logger/logger.dart [hunk 2]
```diff
@@ -1,13 +1,236 @@
...
+    if (shouldLog || shouldSaveInMemory) {
+      DateTime time = DateTime.now();
+      finalMessage =
+          '[${level.name.substring(0, 1).toUpperCase()}][${time.toIso8601String()}] $message';
+    }
+
+    if (finalMessage != null) {
+      if (shouldLog) {
+        // ignore: avoid_print
+        print(finalMessage);
```
Comment (matibat) [inline lib/utils/logger/logger.dart:101]: This one is intended?

**Review by matibat — COMMENTED** [review]
LGTM - will approve after a quick test

**Review by matibat — APPROVED** [review]

File: lib/blocs/pin_file/pin_file_bloc.dart [hunk 3]
```diff
@@ -373,10 +371,9 @@ class PinFileBloc extends Bloc<PinFileEvent, PinFileState> {
     );
   }
 
+// TODO: FIX
```
[the remark this answers is not in the document]
Comment (thiagocarvalhodev) [reply lib/blocs/pin_file/pin_file_bloc.dart:374]: humm, I think was related to some logger, I don't remember, I'll remove it
